egs2/sunda/asr1/run_wld.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+lid=false # whether to use language id as additional label
+
+train_set="train"
```

Comment:
I don't think this shell script is necessary. If you think it is important, please share the reason